src/pages/components/button.mdx

```diff
@@ -61,14 +61,13 @@ import ButtonDangerCode from '!!raw-loader!../../examples/button/ButtonDanger.ts
 Mark a button disabled with the disabled property.
 [Read more about disabled state usage](#todo)
 
-## Best practices
+## Configuration
 
-### Writing button labels
+<Configuration reactPackage={props.data.mdx.package} propSheets={props.data.mdx.propsSheets} />
 
-## Package
+## API
 
-<PackageDescription data={props.data.mdx.package} />
-<PropSheets data={props.data.mdx.propsSheets} reactPackage={props.data.mdx.package} />
+<PropSheets propSheets={props.data.mdx.propsSheets} />
```

Comment:
fwiw, doesn't have to happen in this PR. But we need to tee-up support for `@ignore` as handled by Styleguidist. https://react-styleguidist.js.org/docs/documenting/#ignoring-props. In this prop sheet, we shouldn't see `isSelected`. See https://github.com/zendeskgarden/react-components/blob/992c41ff83f4f6349bd491b6cdbe3dc9373b0fae/packages/buttons/src/elements/Button.tsx#L30-L31

If not already done, please capture this issue. It is a must-do for v1.

Reply:
So we aren't bloating the MVP Story points up and down in Jira adding tasks, I have started collecting bugs, nits and issues like this in issue #1280. I know we will have to address it a different way, but I was capturing it there for now.

Reply:
Excellent. TY